lib/notifier.rb

```diff
@@ -0,0 +1,17 @@
+require 'twilio-ruby'
+class Notifier
+  def initialize(config_loader_klass:)
+    config_loader = config_loader_klass.new('./support/config.yaml')
+    account_sid = config_loader.account_sid
+    auth_token = config_loader.auth_token
+    @client = Twilio::REST::Client.new account_sid, auth_token
```

Comment:
_this_ is what should be extracted out and replaced with a double in your testing.